### PR TITLE
Add UID to log location to allow multi-tenancy.

### PR DESCRIPTION
**Pull request by dspearson**
Depends upon: https://github.com/mopemope/meghanada-server/pull/82

### CODE REVIEW

File: meghanada.el [hunk 1]
```diff
@@ -511,13 +511,14 @@ function."
...
-              (cmd (format "%s %s %s -Dfile.encoding=UTF-8 -jar %s -p %d %s"
+              (cmd (format "%s %s %s -Dfile.encoding=UTF-8 -jar %s -p %d %s %s"
                            (shell-quote-argument meghanada-java-path)
                            (meghanada--server-options)
                            meghanada-server-jvm-option
                            (shell-quote-argument jar)
                            meghanada-port
-                           (if meghanada-debug "-v" "")))
+                           (if meghanada-debug "-v" "")
+                           (concat "-l /tmp/meghanada_server_" (number-to-string (user-uid)) ".log")))
```
Comment (mopemope) [inline meghanada.el]: Please use  `temporary-file-directory` .

Comment (dspearson) [reply meghanada.el]: Sorry! Addressed. My first time with elisp.

**Comment by dspearson** [comment]
I also updated the documentation to reflect this change.

**Comment by mopemope** [comment]
@dspearson Merged. Thanks !

**Pull request merged**